### PR TITLE
Document `influxd upgrade` command

### LINKED ISSUE #1602: Document `influx upgrade` command
Touch base with Russ to document the upgrade command for GA.
How users names/passwords/tokens convert from 1.x

### PR DESCRIPTION
**Pull request by pierwill**
Closes #1602.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/cli/influxd/upgrade/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+weight: 201
+products: [oss]
+---
+
+Use the `influxd upgrade` command to upgrade an instance of InfluxDB 1.x to InfluxDB 2.0.
+This will copy all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
+[retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)(used in 1.x)
+over to [buckets](/influxdb/v2.0/reference/glossary/#bucket) (used in 2.0).
+
+## Usage
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade/_index.md:16]: I don't know where to put this, but somewhere we need a big warning that says they should backup their data before upgrading.

File: content/influxdb/v2.0/reference/cli/influxd/upgrade/v1-dump-meta.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  Output 1.x metadata
+influxdb/v2.0/tags: []
+menu:
+  influxdb_2_0_ref:
+    parent: influxd upgrade
+weight: 301
+products: [oss]
+---
+
+The `influxd upgrade v1-dump-meta` command prints the contents of `meta.db` to the standard output.
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade/v1-dump-meta.md:13]: is there a 1.x command that is similar to this?

File: content/influxdb/v2.0/reference/cli/influxd/upgrade/v2-dump-meta.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  Output 2.0 boltdb metadata
+influxdb/v2.0/tags: []
+menu:
+  influxdb_2_0_ref:
+    parent: influxd upgrade
+weight: 301
+products: [oss]
+---
+
+The `influxd upgrade v2-dump-meta` command prints the contents of `influxdb.bolt` to the standard output.
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade/v2-dump-meta.md:13]: what does this help you do?

Comment (pierwill) [reply content/influxdb/v2.0/reference/cli/influxd/upgrade/v2-dump-meta.md]: Turns out these are developer tools. Removed.

**Pull request merged**